### PR TITLE
fix(chat): fix the infinite loader for "New conversation" (Issue #2358)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

fix the infinite loader for "New conversation"

Issues:

- Issue #2358

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2366.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2366.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by Gimir — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2366.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2366.nightly-test.deltixhub.io

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2366.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2366.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2366.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/store/conversations/conversations.epics.ts [hunk 1]
```diff
@@ -407,6 +407,7 @@ const createNewConversationsEpic: AppEpic = (action$, state$) =>
       }) => {
         return state$.pipe(
           startWith(state$.value),
+          filter((state) => ModelsSelectors.selectIsRecentModelsLoaded(state)),
```
Comment (Alexander-Kezik) [inline apps/chat/src/store/conversations/conversations.epics.ts]:
```suggestion
          filter(ModelsSelectors.selectIsRecentModelsLoaded),
```

I suppose `state` has `any` type in the next `map` if we will return a callback

**Review by Alexander-Kezik — DISMISSED** [review]

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**